src/components/WeeklySummariesReport/WeeklySummariesReport.jsx

```diff
@@ -87,6 +87,8 @@ export class WeeklySummariesReport extends Component {
 
     this.canPutUserProfileImportantInfo = hasPermission('putUserProfileImportantInfo');
     this.bioEditPermission = this.canPutUserProfileImportantInfo;
+    this.canToggleRequestBio = hasPermission('requestBio');
```

Comment:
```suggestion
    this.canToggleRequestBio = props.hasPermission('requestBio');
```
this is for another PR... should rebase your commits relevant to this PR onto dev.